assignments/setup.md

```diff
@@ -277,6 +277,13 @@ Press:
 You should see your patch series including your introduction in the list of messages.
 If you do, congratulations! You successfully completed the setup.
 
```

Comment:
This info is actually incorrect, the email will not show up, it only appears on dashboard (until after the deadline). I should have been clear about this problem in the related issue, but in connection with adding these correct instructions about checking the dashboard, it makes sense to remove this outdated info too.